src/modules/emotion.command.js

```diff
@@ -0,0 +1,23 @@
+const {text, pictures} = require('./emotion-list.json');
+
+// TODO: любая эмоция может быть командой. любая команда может быть эмоцией
+// TODO: получаю эмоцию-команду и беру из пула сообщение и картинку
+const emotionCommand = async function(response, { username, input, i18n }) {
+    if (input[0] !== '!') {
```

Comment:
good

Reply:
Move to config?

Reply:
yep! agree... in config like `SOCIAL_PREFIX`

Reply:
done